src/locale.en.js

```diff
@@ -173,18 +174,92 @@ export const favorites = {
 };
 
 export const counties = {
+    blank: '',
+    adams: 'Adams',
+    alamosa: 'Alamosa',
+    arapahoe: 'Arapahoe',
+    archuleta: 'Archuleta',
+    baca: 'Baca',
+    bent: 'Bent',
+    boulder: 'Boulder',
+    broomfield: 'Broomfield',
     chaffee: 'Chaffee',
+    cheyenne: 'Cheyenne',
+    clear_creek: 'Clear Creek',
+    conejos: 'Conejos',
+    costilla: 'Costilla',
+    crowley: 'Crowley',
+    custer: 'Custer',
+    delta: 'Delta',
+    denver: 'Denver',
+    dolores: 'Dolores',
+    douglas: 'Douglas',
+    eagle: 'Eagle',
+    el_paso: 'El Paso',
+    elbert: 'Elbert',
+    fremont: 'Fremont',
+    garfield: 'Garfield',
+    gilpin: 'Gilpin',
+    grand: 'Grand',
+    gunnison: 'Gunnison',
+    hinsdale: 'Hinsdale',
+    huerfano: 'Huerfano',
+    jackson: 'Jackson',
+    jefferson: 'Jefferson',
+    kiowa: 'Kiowa',
+    kit_carson: 'Kit Carson',
     lake: 'Lake',
-    el_paso: 'El Paso'
+    la_plata: 'La Plata',
+    larimer: 'Larimer',
+    las_animas: 'Las Animas',
+    lincoln: 'Lincoln',
+    logan: 'Logan',
+    mesa: 'Mesa',
+    mineral: 'Mineral',
+    moffat: 'Moffat',
+    montezuma: 'Montezuma',
+    montrose: 'Montrose',
+    morgan: 'Morgan',
+    otero: 'Otero',
+    ouray: 'Ouray',
+    park: 'Park',
+    phillips: 'Phillips',
+    pitkin: 'Pitkin',
+    prowers: 'Prowers',
+    pueblo: 'Pueblo',
+    rio_blanco: 'Rio Blanco',
+    rio_grande: 'Rio Grande',
+    routt: 'Routt',
+    saguache: 'Saguache',
+    san_juan: 'San Juan',
+    san_miguel: 'San Miguel',
+    sedgwick: 'Sedgwick',
+    summit: 'Summit',
+    teller: 'Teller',
+    washington: 'Washington',
+    weld: 'Weld',
+    yuma: 'Yuma'
```

Comment:
Even though some of these will never have campsites, I got tired of having to add one at a time after I find a site there. Just added all counties in CO for now, and we can trim them back down later if desired.